src/main/services/steam.service.ts

```diff
@@ -25,18 +26,9 @@ export class SteamService{
     return SteamService.instance;
   }
 
-    public async getActiveUser(): Promise<number>{
-        const res = await regedit.promisified.list(["HKCU\\Software\\Valve\\Steam\\ActiveProcess"]);
-        const keys = res?.["HKCU\\Software\\Valve\\Steam\\ActiveProcess"];
-
-        if(!keys?.exists){ throw "Key \"HKCU\\Software\\Valve\\Steam\\ActiveProcess\" not exist"; }
-
-        return (keys.values?.ActiveUser.value || undefined) as number;
-    }
-
-    public steamRunning(): Promise<boolean>{
-        return this.getActiveUser()
-            .then(userId => !!userId)
```

Comment:
I think it should be preferable to stick with getActiveUser on windows cause some users don't have steam in there tasklist for some reason even if steam is running. And at least getActiveUser handle if the user is not logged